data/World/Fire Temple.json

```diff
@@ -34,6 +36,12 @@
         "locations": {
             "Fire Temple Big Lava Room Lower Open Door Chest": "True",
             "Fire Temple Big Lava Room Blocked Door Chest": "is_adult and has_explosives",
+            "Fire Temple Elevator Room Recovery Heart 1": "(Small_Key_Fire_Temple, 3) and can_use(Goron_Tunic)",
+            "Fire Temple Elevator Room Recovery Heart 2": "(Small_Key_Fire_Temple, 3) and can_use(Goron_Tunic)",
+            "Fire Temple Elevator Room Recovery Heart 3": "(Small_Key_Fire_Temple, 3) and can_use(Goron_Tunic)",
+            "Fire Temple Big Lava Room Pot 1" : "True",
+            "Fire Temple Big Lava Room Pot 2" : "True",
+            "Fire Temple Big Lava Room Pot 3" : "True",
```

Comment:
Getting these in OHKO under FTR is pretty borderline. Personally I'd leave it as "True" still though. (In particular I had trouble getting the recovery heart pot as child with no additional items since I had to wait for the recovery heart to slowly float down.)

Reply:
Drops are all converted to use the rupee mechanic when they are shuffled (they don't slowly float down). I think these should stay True.

Reply:
Big Lava Room already checks for tunic/FTR on room access. I'm talking about whether getting these pots is reasonable under FTR.

Reply:
O gotcha. Well I guess I think it's pretty reasonable under FTR. I've had no issues getting them in 24 seconds

Reply:
You also have to remember you might be on ohko with no items at all. Imagine you're child, can't touch any lava, and have to throw pots to break them.